neqo-transport/src/streams.rs

```diff
@@ -386,15 +425,16 @@ impl Streams {
                     StreamType::BiDi => tparams::INITIAL_MAX_STREAM_DATA_BIDI_REMOTE,
                 };
                 let send_limit = self.tps.borrow().remote().get_integer(send_limit_tp);
-                self.send.insert(
-                    new_id,
-                    SendStream::new(
+		let stream = SendStream::new(
                         new_id,
                         send_limit,
                         Rc::clone(&self.sender_fc),
                         self.events.clone(),
-                    ),
+                    );
+                self.send.insert(
+                    new_id, stream,
                 );
```

Comment:
Is this really what cargo fmt produces?  I'm surprised that it's breaking across lines like this.